### ISSUE TITLE
Interacting with non-standard DataRefs (e.g. Carenado)

### ISSUE THREAD

**Issue by aitecc** [condac/openSimIO#43]
I'm developing an hardware interface to reflect a Carenado Aircraft, that is using non standard datarefs. In FlyWithLua I can use these Datarefs, there is a possibility or workaround to use them also with OpenSimIO?

Example of non standard Carenado Dataref: com/dkmp/BT3[81] (switch toggle position).

**Comment by condac** [comment]
Yes it is possible, but I think I removed that feature very recent because I thought it was related to another crash. 
You can also try and use the option read config in the menu after the aircraft is loaded because then it will find new datarefs that the aircraft create after opensimIO is loaded at the start of the game.
